### PR TITLE
Fixed and add missing translations

### PR DESCRIPTION
**Pull request by crash21**
Fixed and and missing translations for Greek language

### CODE REVIEW

File: Resources/translations/FOSUserBundle.el.yml [hunk 1]
```diff
@@ -36,10 +40,21 @@ registration:
...
     email:
         subject: 'Καλώς ήρθες %username%!'
-        message: "Αγαπητέ %username%!\n\n  Για να ολοκληρώσετε την ενεργοποίηση του λογαριασμού σας - επισκεφθείτε %confirmationUrl%\n\nΕυχαριστούμε πολύ.\n"
+        message: |
+                Αγαπητέ %username%!
+
+                Για να ολοκληρώσετε την ενεργοποίηση του λογαριασμού σας - επισκεφθείτε %confirmationUrl%
+
+                Ευχαριστούμε πολύ."
+
```
Comment (XWB) [inline Resources/translations/FOSUserBundle.el.yml]: This duplicate empty line can be removed.

File: Resources/translations/FOSUserBundle.el.yml [hunk 2]
```diff
@@ -49,12 +64,21 @@ resetting:
...
     email:
         subject: 'Επαναφορά του κωδικού πρόσβασης'
-        message: "Αγαπητέ %username%!\n\n  Για να επαναφέρετε τον κωδικό πρόσβασής σας - επισκεφθείτε %confirmationUrl%\n\nΕυχαριστούμε πολυ.\n"
+        message: |
+                  Αγαπητέ %username%!
+
+                  Για να επαναφέρετε τον κωδικό πρόσβασής σας - επισκεφθείτε %confirmationUrl%
+
+                  Ευχαριστούμε πολυ."
+
```
Comment (XWB) [inline Resources/translations/FOSUserBundle.el.yml]: This duplicate empty line can be removed.

**Comment by crash21** [comment]
hey I leave this line breaks to be identical with the English version for easier compare in case new translation will be added.

**Comment by XWB** [comment]
One empty line is fine, there's no reason to have two empty lines.

**Pull request merged**